TOCrop-iOS/TOCropViewController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Threading.Tasks;
+using UIKit;
+
+namespace TOCrop
+{
+    public partial class TOCropViewController
+    {
+        /// <summary>
+        /// Shows the crop view async.
+        /// </summary>
+        /// <returns>The crop view async.</returns>
```

Comment:
Returns a byte array of the cropped photo or throws an OperationCancelledException if the user cancelled.